lib/commands/deploy.js

```diff
@@ -15,7 +15,15 @@ module.exports = {
 
   run: function(commandOptions, rawArgs) {
     commandOptions.deployTarget = rawArgs.shift();
-    this.ui.verbose = commandOptions.verbose;
+
+    var hasVerboseSetting = this.settings && this.settings['ember-cli-deploy'] &&
+      this.settings['ember-cli-deploy'].verbose;
+    if (hasVerboseSetting && commandOptions.verbose === undefined) {
+      this.ui.verbose = this.settings['ember-cli-deploy'].verbose;
+    } else {
+      this.ui.verbose = commandOptions.verbose;
+    }
+
```

Comment:
@lukemelia Your phrasing would change the code's behavior slightly in that it assumes that the `.ember-cli` setting is `true` rather than passing through the setting value (which I suppose would be ok as long as the default remains `false` and the setting is only to override it).

I was attempting to check for (1) the existence of the setting value and (2) the absence of an explicit flag before (3) assigning the actual value of the setting. If you prefer the initial assignment to occur before the conditional, maybe this instead?
```javascript
this.ui.verbose = commandOptions.verbose;
if (this.ui.verbose === undefined && this.settings && this.settings['ember-cli-deploy'] && this.settings['ember-cli-deploy'].verbose) {
  this.ui.verbose = this.settings['ember-cli-deploy'].verbose;
}
```
It's a long conditional, but it retains my original functionality.